CHANGELOG.md

```diff
@@ -42,6 +54,195 @@ class Pet(Enum):
 Contributed by Terence Honles in PR [17207](https://github.com/python/mypy/pull/17207) and
 Shantanu Jain in PR [18068](https://github.com/python/mypy/pull/18068).
 
+### Added support for @deprecated decorator (PEP 702)
+
+Mypy can now issue errors or notes when code imports a deprecated feature
+explicitly with a `from mod import depr` statement, uses a deprecated feature
+imported otherwise or defined locally. Features are considered deprecated when
+decorated with `warnings.deprecated`, as specified in [PEP 702](https://peps.python.org/pep-0702).
+
+You can enable the error code via `--enable-error-code=deprecated` on the mypy
+command line or `enable_error_code = deprecated` in the mypy config file.
+Use the option `--report-deprecated-as-note` to turn all such errors into notes.
```

Comment:
can this also be specified in the config file?

Reply:
Yes, it can also be specified via `report_deprecated_as_note = True` I can add this as well in the text here.